src/sage/rings/finite_rings/integer_mod.pyx

```diff
@@ -1977,8 +1979,8 @@ cdef class IntegerMod_gmp(IntegerMod_abstract):
         r"""
         EXAMPLES::
 
-            sage: p = next_prime(2^32)
-            sage: GF(p)(int(p+1))
+            sage: p = next_prime(2^32)                                                  # optional - sage.libs.pari
+            sage: GF(p)(int(p + 1))                                                     # optional - sage.rings.finite_rings
```

Comment:
Does `sage.rings.finite_rings` depend on `sage.libs.pari`? If not, line 1983 needs also `sage.libs.pari`?

Reply:
Yes, more specifically, it requires `sage.rings.finite_rings.element_pari_ffelt`